src/main/java/lottogame/controller/GameController.java

```diff
@@ -1,28 +1,44 @@
 package lottogame.controller;
 
 import lottogame.domain.LottoMachine;
-import lottogame.domain.dto.LottoResult;
-import lottogame.domain.dto.LottoResults;
+import lottogame.domain.TicketMachine;
+import lottogame.domain.dto.LottoDto;
+import lottogame.domain.dto.LottoResultDto;
+import lottogame.domain.lotto.Lotto;
+import lottogame.domain.statistic.LottoResult;
+import lottogame.domain.statistic.LottoResults;
 import lottogame.domain.lotto.WinningLotto;
 import lottogame.view.InputView;
 import lottogame.domain.lotto.Lottos;
 import lottogame.domain.Money;
 import lottogame.view.OutputView;
 
 import java.util.List;
+import java.util.Scanner;
 
 public class GameController {
+    private static final InputView inputView = new InputView(new Scanner(System.in));
+    private static final LottoMachine lottoMachine = new LottoMachine();
+    private static final TicketMachine ticketMachine = new TicketMachine();
 
     public void run() {
-        LottoMachine lottoMachine = new LottoMachine();
-        Money money = new Money(InputView.inputMoney());
-        int quantity = lottoMachine.purchaseQuantity(money);
-        Lottos lottos = new Lottos(lottoMachine.buyLotto(quantity));
-        OutputView.showLottos(lottos.numbersOfLottos());
-        WinningLotto winningLotto = askWinningLotto();
-        LottoResults lottoResults = matchLottos(lottos, winningLotto);
-        lottoResults.makeStatistics(money);
-        OutputView.printResult(lottoResults);
+        Money money = new Money(inputView.inputMoney());
+        int manualQuantity = inputView.inputManualQuantity();
+        money = ticketMachine.buyManualTicket(money, manualQuantity);
+        int autoTicketQuantity = ticketMachine.buyableAutoTicketQuantity(money);
+        List<Lotto> manualLottos = makeManualLotto(autoTicketQuantity);
+        List<Lotto> autoLottos = lottoMachine.buyAutoTicket(autoTicketQuantity);
+        Lottos lottos = new Lottos(manualLottos, autoLottos);
+        OutputView.showLottos(manualQuantity, lottos.numbersOfLottos());
+        LottoResults lottoResults = matchLottos(lottos, askWinningLotto());
+        LottoResultDto lottoResultDto = lottoResults.makeStatistics(money);
+        OutputView.printResult(lottoResultDto);
+    }
+
+    private List<Lotto> makeManualLotto(int autoTicketQuantity) {
+        List<LottoDto> manualLotto = inputView.inputManualLotto(autoTicketQuantity);
+        List<Lotto> manualLottos = lottoMachine.makeManualLotto(manualLotto);
+        return manualLottos;
```

Comment:
```suggestion
        return lottoMachine.makeManualLotto(manualLotto);
```
바로 return 해줘도 좋을 것 같습니다 :)

Reply:
오호,, 가독성을 높여주려고 저렇게 설정해보았어요!
저렇게 되면 manualLottos라는 인스턴스가 생성되기 때문에 굳이 저렇게 쓸 필요 없다는 말씀이시겠지요..? 🤔🤔
수정하여 적용했습니다. 🥰

Reply:
사람마다 다르겠지만, 어짜피 메서드명에 명시되어 있으니 변수를 안 만들어줘도 의미가 잘 전달될거 같아요~ㅎㅎ